### ISSUE TITLE
Document lack of support for recording direct to local SD card

### ISSUE THREAD

**Issue by acbox** [martinbutt/openmiko#12]
**Is your feature request related to a problem? Please describe.**
Some users expect local recording to work out of the box as opposed to focusing on exposing a stream for off-device recording.

**Describe the solution you'd like**
Update the README to clearly state that it's not a supported feature.

**Describe alternatives you've considered**
N/A

**Additional context**
If anyone knows how to get this working, please share. I haven't pushed a PR myself to update the docs as I'm not 100% sure it is accurate to say local recording is deliberately not supported. Happy to do so if it's true.

Thanks for sharing your work on keeping this brilliant project alive.

**Comment by martinbutt** [comment]
It's not well documented, but local recording is supported. Set `ENABLE_RECORDING=1` in your `openmiko.conf` and your recordings will appear on the SD card in `/recordings/`. Can you let me know if that works for you and I can update the docs?

**Comment by acbox** [comment]
Thanks for responding. I've gone back to the stock Wyze firmware to use with [docker-wyze-bridge](https://github.com/mrlt8/docker-wyze-bridge) and Frigate, so I can't test setting `ENABLE_RECORDING=1` in the config. I think I tried it before and it didn't record anything. In general I found openmiko very opaque and difficult to work with. My main problem was v4l2rtspserver hanging silently all the time, requiring a reboot. Thought it might be my cheap SD card, though it works fine for recording on the stock Wyze firmware. Thanks again.
